src/middlewared/middlewared/plugins/chart_releases_linux/chart_release.py

```diff
@@ -458,14 +459,23 @@ async def do_update(self, job, chart_release, data):
         job.set_progress(100, 'Update completed for chart release')
         return await self.get_instance(chart_release)
 
-    @accepts(Str('release_name'))
+    @accepts(
+        Str('release_name'),
+        Dict(
+            'options',
+            Int('wait_secs', default=300)
```

Comment:
Can `timeout` be a better commonly used name?

Reply:
Agreed